tests/test-generators.py

```diff
@@ -73,6 +73,16 @@ def parse_output(result, error_code):
             limit = 9
         elif "negative-reserve-fee.clvm" in g:
             limit = 4
+        elif "block-834752" in g:
```

Comment:
Might want to (in future) just make this a list of exceptions that you iterate through. Might be a bit cleaner.

Reply:
Something like

```python
TEST_LIMITS = [("duplicate-coin-announce", 9), ("negative-reserve-fee", 4), ...]
...
for k, v in TEST_LIMITS:
    if k in g:
        limit = v
        break
```

Reply:
Yeah, I'm hoping to improve the performance to make all these run in less than 1.5s. Maybe if this list gets longer before that happens...